lib/response/CallResponse.js

```diff
@@ -1,13 +1,7 @@
-var Rx = require("rx/dist/rx") && require("rx/dist/rx.aggregates");
```

Comment:
Yes, before I remove that I thought that maybe this is required for webpack load the rx files. But after I saw that the Rx is only used to load ```Observable``` and ```CompositeDisposable``` and CallResponse is not a Subclass of Observable anymore due to #552.